cmd/tuf/app/init.go

```diff
@@ -86,26 +86,18 @@ func InitCmd(directory string, targets targetsFlag) error {
 	if err != nil {
 		return err
 	}
-	expiration := time.Now().AddDate(0, 6, 0)
+	expiration := time.Now().AddDate(0, 6, 0).UTC()
 	relativePaths := []string{}
-	root.Expires = expiration
-	root.Version++
```

Comment:
I don't see this version increment moved somewhere else, is it not needed?

Reply:
Yep, it's incremented in AddVerification...

Reply:
Ah, now I see it, thanks